src/pluggy/_tracing.py

```diff
@@ -1,19 +1,18 @@
 """
 Tracing utils
 """
-from .callers import _Result
 
 
 class TagTracer(object):
     def __init__(self):
-        self._tag2proc = {}
-        self.writer = None
+        self._tags2proc = {}
+        self._writer = None
```

Comment:
as far as i can tell the "writer" is a "print" style function we ight be able to simplify